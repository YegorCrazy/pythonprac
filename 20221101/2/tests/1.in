a = Triangle((0, 0), (1, 1), (2, 0))
b = Triangle((4, 2), (1, 3), (0, 3))
if a:
	print(round(abs(a), 3))
if b:
	print(round(abs(b), 3))
print(a < b)
print(a in b, b in a, sep = ', ')
print(a & b)
