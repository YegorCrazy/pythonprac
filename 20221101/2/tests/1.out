1.0
0.5
False
False, False
False
